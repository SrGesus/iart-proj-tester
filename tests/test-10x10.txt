FD	FD	VE	LH	LV	VB	VC	LV	FC	FC
BD	BD	LH	FD	VD	LH	LV	VD	LH	VD
FE	BC	VD	FB	BC	LH	BB	BB	FE	FB
FB	VE	FE	BB	BC	VD	BB	LH	LV	VD
VD	FE	FD	BD	FB	FD	BE	LH	FD	FE
LV	FB	LV	BD	LH	BD	BC	LH	LH	VB
LV	LH	BE	BD	VB	VE	VD	LH	FE	FB
VC	BE	VE	BD	BD	VD	BE	VD	FD	LV
FD	FD	FD	VC	LV	FC	FB	LH	FD	LH
VB	BD	LH	LV	BB	BB	LV	BE	BD	VB
