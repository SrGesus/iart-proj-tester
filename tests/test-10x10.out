FB	FB	VB	LH	LH	VE	VB	LH	FE	FB
BD	BE	LV	FD	VE	LV	LV	VB	LH	VC
FC	BD	VC	FD	BE	LV	BD	BC	FE	FB
FD	VC	FD	BB	BC	VC	BD	LH	LH	VC
VB	FE	FB	BD	FE	FB	BD	LH	FE	FB
LV	FB	LV	BD	LH	BC	BC	LH	LH	VC
LV	LV	BD	BE	VB	VE	VB	LH	FE	FB
VD	BC	VC	BD	BE	VD	BE	VB	FE	LV
FB	FB	FD	VC	LV	FB	FC	LV	FB	LV
VD	BC	LH	LH	BC	BC	LH	BC	BC	VC
